src/rdbms/mongoose_rdbms_odbc.erl

```diff
@@ -184,11 +184,16 @@ unsplit_query([QueryHead], _ParamMappers, [], QueryAcc, ParamsAcc) ->
 unsplit_query([QueryHead | QueryRest], ParamMappers, [Param | Params], QueryAcc, ParamsAcc) ->
     {{value, Mapper}, ParamMappersTail} = queue:out(ParamMappers),
     NextParamMappers = queue:in(Mapper, ParamMappersTail),
-    {InlineQuery, ODBCParam} = Mapper(Param),
```

Comment:
Why this line was invalid?

Reply:
It was invalid because it didn't respect the `null` value which in some cases is needed in our prepared queries and should not be mapped.